### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by ngsonha**
Admin has full access to the website and can read/ write/ update/ delete any of the posts and Anyone can read these posts

### CODE REVIEW

File: app/User.php [hunk 1]
```diff
@@ -52,5 +52,12 @@ public function can_post()
     return false;
   }
 
-  
+  public function is_admin()
+  {
+    $role = $this->role;
+    if ($role == 'admin') {
```
Comment (trphuongnam) [inline app/User.php]: role sử dụng enums nhé

File: resources/views/admin/profile.blade.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    </li>
+    <li class="list-group-item panel-body">
+      <table class="table-padding">
+        <style>
+          .table-padding td {
+            padding: 3px 8px;
+          }
+        </style>
+        <tr>
+          <td>Total Posts</td>
```
Comment (trphuongnam) [inline resources/views/admin/profile.blade.php]: Sử dụng i18n nhé

File: app/User.php (hunk 1 above)
Comment (ngsonha) [reply app/User.php]: ddax fix

File: resources/views/admin/profile.blade.php (hunk 2 above)
Comment (ngsonha) [reply resources/views/admin/profile.blade.php]: da fix

**Pull request merged**